client/modules/IDE/actions/projects.js

```diff
@@ -57,5 +56,5 @@ export function deleteProject(id) {
 }
 
 export function sortProjects(order, orderBy) {
-  return () => store.dispatch({ type: ActionTypes.SET_SORT_PARAM, payload: { orderBy, order } });
+  return dispatch => dispatch({ type: ActionTypes.SET_SORT_PARAM, payload: { orderBy, order } });
```

Comment:
because the action isn't asynchronous, the action doesn't need to return a function. it can simply return an object, and you can bind it to dispatch using `bindActionCreators`/`mapDispatchToProps`